src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -345,6 +348,11 @@ func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			handler.EnqueueRequestsFromMapFunc(r.reconcileClusterForPods),
 			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{}),
 		).
+		Watches(
+			&source.Kind{Type: &corev1.Secret{}},
+			handler.EnqueueRequestsFromMapFunc(r.reconcileClusterForExternalCASecret),
+			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{}),
+		).
```

Comment:
Is this necessary? As I have had it explained to me, seastar watches for changes to the certificates and reloads them automatically.